bundle/compliance/cis_azure/rules/cis_7_3/rule.rego

```diff
@@ -0,0 +1,16 @@
+package compliance.cis_azure.rules.cis_7_3
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import data.compliance.policy.azure.disk.ensure_encryption as audit
+
+finding = result {
+	# filter
+	data_adapter.is_attached_disk
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(audit.is_encryption_enabled),
```

Comment:
I can't find the audit function

Reply:
It's an import:

```
import data.compliance.policy.azure.disk.ensure_encryption as audit
```

Reply:
Which is shared with rule 7.4 (already committed)